src/components/Dropdown/Dropdown.tsx

```diff
@@ -308,19 +361,44 @@ export default class Dropdown extends AutoControlledComponent<
   private renderItemsList(
     styles: ComponentSlotStylesInput,
     variables: ComponentVariablesInput,
-    getMenuProps: (options?: GetMenuPropsOptions, otherOptions?: GetPropsCommonOptions) => any,
-    getItemProps: (options: GetItemPropsOptions<ShorthandValue>) => any,
     isOpen: boolean,
     highlightedIndex: number,
+    toggleMenu: () => void,
+    selectItemAtIndex: (index: number) => void,
+    getMenuProps: (options?: GetMenuPropsOptions, otherOptions?: GetPropsCommonOptions) => any,
+    getItemProps: (options: GetItemPropsOptions<ShorthandValue>) => any,
+    getInputProps: (options?: GetInputPropsOptions) => any,
   ) {
-    const accessibilityMenuProps = getMenuProps({ refKey: 'innerRef' })
+    const accessibilityMenuProps = {
+      ...getMenuProps({ refKey: 'innerRef' }, { suppressRefError: true }),
+    }
+    if (!this.props.search) {
```

Comment:
if we have a non-search, the keyboard handles, aria-activedescendant and some other stuff that was on the SearchInput now goes on the List. Also there were some custom handlers added on the List, in `this.handleListKeyDown`. Needed these custom handlers because Downshift does not support non-search by default.